applications/Chat/evaluate/metric.py

```diff
@@ -0,0 +1,152 @@
+import jieba
+from nltk.translate.bleu_score import sentence_bleu
+from rouge_chinese import Rouge
```

Comment:
If this is only for Chinese one, your naming should show that this is Chinese Rouge. Please add alias for this import.

Reply:
from rouge_chinese import Rouge as Rouge_cn